iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/pipe/task/CreatePipeProcedureV2.java

```diff
@@ -203,7 +203,9 @@ public void executeFromCalculateInfoForTask(ConfigNodeProcedureEnv env) {
     }
 
     pipeRuntimeMeta = new PipeRuntimeMeta(consensusGroupIdToTaskMetaMap);
-    pipeRuntimeMeta.getStatus().set(PipeStatus.RUNNING);
+    if (!createPipeRequest.needManuallyStart) {
```

Comment:
Can needManuallyStart be null?

Reply:
should check issetxxx? or you should relay on the default value for boolean

Reply:
yes, we rely on the default `false` value here

Reply:
I'm afraid it's of type Boolean.. not boolean.

Reply:
<img width="845" alt="image" src="https://github.com/user-attachments/assets/c0ac1bf2-39c9-4552-a400-eccfdb7cae58">
I have tested it, it's a primitive boolean type with default `false` value. And pipe it all passed, which indicates its correctness as well.

Reply:
It's boolean, so the logic here is right. @SteveYurongSu 
<img width="1210" alt="image" src="https://github.com/user-attachments/assets/9ec172aa-2f66-4826-aec3-146d3a63c6a2">